src/components/views/messages/MessageActionBar.js

```diff
@@ -170,6 +173,17 @@ export default class MessageActionBar extends React.PureComponent {
         });
     };
 
+    onThreadClick = () => {
```

Comment:
public/private/etc

Reply:
This one's a JS file, haven't come round to migrating it yet